content/en/docs/Core concepts/concurrency.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Tuning concurrency"
+date: 2020-11-02
+weight: 3
+---
+
+Baker filters and ouput can be configured with a high concurrency profile (input doesn't support
+parallel processing).
+
+This means that those components will process Records in parallel during the pipeline,
+improving performances but adding a small cons: Records order isn't guaranteed anymore
```

Comment:
use of future tense.

Also, this seems a bit marketing, but I'd  prefer if we say that by default records are concurrently processed, without any ordering guarantee. However **you can** guarantee ordering by making the whole pipeline synchronous (procs=1 everywhere)